Compilation fail : rawprepare.c:417:11: error: ‘d’ is predetermined ‘shared’ for ‘shared’ shared(d) schedule(static)

**Describe the bug/issue**

Buil stops with the following error : In file included from /home/pierre/darktable/build/lib/darktable/plugins/introspection_rawprepare.c:55:
/home/pierre/darktable/src/iop/rawprepare.c: In function ‘process’:
/home/pierre/darktable/src/iop/rawprepare.c:417:11: error: ‘d’ is predetermined ‘shared’ for ‘shared’
     shared(d) schedule(static)


**To Reproduce**

Follow instruction on the developper's wiki to compile Darktable.
./build.sh  
build stops at 73% with an error

**Expected behavior**

Compilation success.

**Platform**
_Please fill as much information as possible in the list given below. Please state "unknown" where you do not know the answer and remove any sections that are not applicable _

* darktable version : 3.8.1
* OS                :    4.19.0-20-amd64 #1 SMP Debian 4.19.235-1 (2022-03-17) x86_64 GNU/Linux
* Linux - Distro    : Debian 10 (oldstable)
* Memory            :  32 Go
* Graphics card     : 00:02.0 VGA compatible controller: Intel Corporation HD Graphics 530 (rev 06)
(Main) +  01:00.0 VGA compatible controller: NVIDIA Corporation GK104 [GeForce GTX 770] (rev a1)
(openCL only, no screen plugged)
* Graphics driver   : kernel for Intel and Nvidia 418.211.00-1 
* OpenCL installed  : yes
* OpenCL activated  : yes
* Xorg              :  7.7+19
* Desktop           : Gnome 5.5.1-2
* GTK+              :  3.24.5
* gcc               :  8.3.0-6
* cflags            :  ?
* 
* CMAKE_BUILD_TYPE  :  ?

[build.log](https://github.com/darktable-org/darktable/files/8743616/build.log)

--- a/src/iop/rawprepare.c
+++ b/src/iop/rawprepare.c
@@ -414,7 +414,7 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const 
 #pragma omp parallel for default(none) \
     dt_omp_firstprivate(csx, csy, roi_out, out, im_to_rel_x, im_to_rel_y, rel_to_map_x, rel_to_map_y, \
                         map_w, map_h, map_origin_h, map_origin_v) \
-    shared(d) schedule(static)
+    dt_omp_sharedconst(d) schedule(static)
 #endif
     for(int j = 0; j < roi_out->height; j++)
     {

